sirepo/job.py

```diff
@@ -102,6 +105,21 @@
 #: categories of jobs
 KINDS = frozenset((SEQUENTIAL, PARALLEL))
 
+#: Max values for each sbatch queue
+SBATCH_QUEUE_MAXES = PKDict(
```

Comment:
SBATCH_QUEUE_MAXES => NERSC_QUEUE_MAX. I don't like "MAXES".  LIMITS might be ok, but MAX is sufficient. Plural is unnecessary.

Reply:
https://github.com/radiasoft/sirepo/pull/2511/commits/779ddedc7bc17f3c385612be4fee7c6b6a503204